topics/climate/tutorials/pangeo/tutorial.md

```diff
@@ -0,0 +1,556 @@
+
+*Our mission is to cultivate an ecosystem in which the next generation of open-source analysis tools for ocean, atmosphere and climate science can be developed, distributed, and sustained. These tools must be scalable in order to meet the current and future challenges of big data, and these solutions should leverage the existing expertise outside of the geoscience community.*
+
+In this tutorial, you will learn how to manipulate [netCDF](https://en.wikipedia.org/wiki/NetCDF) data files using [Xarray](https://xarray.pydata.org/en/stable/) Galaxy Tools. 
+
+
+> ### {% icon comment %} Xarray and Earth Science
+> Xarray works with labelled multi-dimensional arrays and can be used for a very wide range of data and data formats. In this training material, we focus on the usage of Xarray for Earth Science data following the [CF-Convention](https://cfconventions.org/). However, some Galaxy Tools also work for non Earth Science datasets and if needed current Xarray Galaxy Tools could be extended to accomodate new usage.
+{: .comment}
+
+In this tutorial, we will be using data from [Copernicus Atmosphere Monitoring Service](https://ads.atmosphere.copernicus.eu/)
+and more precisely PM2.5 ([Particle Matter < 2.5 μm](https://en.wikipedia.org/wiki/Particulates#Size,_shape_and_solubility_matter)) 4 days forecast from December, 22 2021. This dataset is very small and there is no need to parallelize our data analysis. Parallel data analysis with Pangeo is not covered in this tutorial. 
+
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Create a history
+
+> ### {% icon hands_on %} Hands-on: Create history
+>
+> 1. Make sure you start from an empty analysis history.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. **Rename your history** to be meaningful and easy to find. For instance, you can choose **Pangeo 101 for everyone - Xarray** as the name of your new history.
+>
+>    {% snippet faqs/galaxy/histories_rename.md %}
+>
+{: .hands_on}
+
+
+##  Upload CAMS PM2.5 data
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Import the files from [Zenodo]({{ page.zenodo_link }}) or from
+>    the shared data library (`GTN - Material` -> `{{ page.topic_name }}`
+>     -> `{{ page.title }}`):
+>
+>    ```
+>    https://zenodo.org/record/5805953/files/CAMS-PM2_5-20211222.netcdf
+>    ```
+>
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+> 2. If needed rename the datasets to `CAMS-PM2_5-20211222.netcdf`
+> 3. Check that the datatype is `netcdf`
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+> 4. Add a tag corresponding to ads (for Atmosphere Data Service)
+>
+>    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>
+{: .hands_on}
+
+
+# Understanding our dataset
+
+
```

Comment:
I would include some additional information from the  [CAMS European air quality forecast webpage](https://ads.atmosphere.copernicus.eu/cdsapp#!/dataset/cams-europe-air-quality-forecasts?tab=overview) website in this section. I consider this training really interesting, but I think it would be great to include basic information to contextualize each of the steps, as the information contained in the recommended sources of information is perhaps too technical for newcomers.